### PR TITLE
added PV collections

### PR DESCRIPTION
**Pull request by mattballard-planet**
added PV pages and subpages for open data collections

### CODE REVIEW

**Comment by mattballard-planet** [comment]
Hi @Dorothyrono, @gmilcinski asked me to connect with you about reviewing this request to add Planetary Variables here, can you review, please?

**Comment by Dorothyrono** [comment]
on it

**Review by Dorothyrono — CHANGES_REQUESTED** [review]

File: collections/crop-biomass.yaml [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+  This enables better-informed decisions, input optimization, interventions, and harvest planning. 
+  This is a relative measure of biomass, so each pixel value has a value of 0 (low biomass) to 1 (high biomass). 
+  The product is provided daily and every day, the user gets a new raster with the data from the most updated direct observation from the day before.
+AdditionalInfoExternal:
+    Title: Open Data for Crop Biomass
+    Path: crop-biomass/README.MD
+Documentation: "[here](https://docs.sentinel-hub.com/api/latest/data/planetary-variables/crop-biomass/)"
+Image: crop-biomass/crop-biomass.png
+Resolution: 10m
+GeographicalCoverage: Global; Open data in [limited areas](crop-biomass/README.MD);
```
[the remark this answers is not in the document]
Comment (mattballard-planet) [reply collections/crop-biomass.yaml:13]: I was modelling off of this example: https://github.com/sentinel-hub/public-collections/blob/main/collections/vegetation-phenology-and-productivity-parameters-season-2.yaml

which appears to refer to a markdown file. 

I can absolutely change these to readme.html, but just want to double check before I make the change.

also, can the file be named anything? like open-collection.md or open-collection.html?

File: collections/crop-biomass.yaml [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+  - satellite imagery
+  - planetary variables
+  - agriculture
+  - biomass
+License: |
+  [License](https://www.planet.com/licensing-information/)
+Resources:
+  - EndPoint: services.sentinel-hub.com
+    Type: BYOC collection
+    Notes: Contains the data purchased by individual user. [Explore sample CC-BY-SA open data here](crop-biomass/README.MD).
```
[the remark this answers is not in the document]
Comment (mattballard-planet) [reply collections/crop-biomass.yaml:32]: same comment as before

File: collections/land-surface-temperature.yaml [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+Description: |
+  [Land Surface Temperature](https://www.planet.com/products/planetary-variables/) provides twice-daily measurements of land surface temperature with high spatial resolution and consistency. 
+  LST data eliminates the need for maintaining large networks of physical sensors, reduces the impact of cloud cover on measurements, and enables a more accurate reflection of ground conditions across large areas.
+AdditionalInfoExternal:
+    Title: Open Data for Land Surface Temperature
+    Path: land-surface-temperature/README.MD
+Documentation: "[here](https://docs.sentinel-hub.com/api/latest/data/planetary-variables/land-surface-temp/)"
+Image: land-surface-temperature/land-surface-temperature.png
+Resolution: 100m and 1000m
+GeographicalCoverage: Global; Open data in [limited areas](land-surface-temperature/README.MD);
```
[the remark this answers is not in the document]
Comment (mattballard-planet) [reply collections/land-surface-temperature.yaml:11]: same response as earlier, will adjust after that discussion

File: collections/crop-biomass.yaml (hunk 1 above)
[the remark this answers is not in the document]
Comment (Dorothyrono) [reply collections/crop-biomass.yaml:13]: Hey Matt,

> I was modelling off of this example: https://github.com/sentinel-hub/public-collections/blob/main/collections/vegetation-phenology-and-productivity-parameters-season-2.yaml
> 
> which appears to refer to a markdown file

It depends on the template

Right, the `AdditionalInfoExternal` refers to a mark down file , as it has a function to compile the markdown file to a html page. quick check [here](https://github.com/sentinel-hub/public-collections/blob/main/gulpfile.js#L291)

The other templates use other functions; in this case `GeographicalCoverage ` template for example takes in simple markdown text . That's why a direct link to the page would work

Same to the `notes` template

[the remark this answers is not in the document]
Comment (Dorothyrono) [reply collections/crop-biomass.yaml:13]: > also, can the file be named anything? like open-collection.md or open-collection.html?

Yes,  just make it consistent.   

You would have something like this 
```
AdditionalInfoExternal:
    Title: Open Data for Crop Biomass
    Path: crop-biomass/open-collection.md
----
GeographicalCoverage: Global; Open data in [limited areas](open-collection.html); 
Notes: Contains the data purchased by individual user. [Explore sample CC-BY-SA open data here](open-collection.html).
```

Basically, the absolute link is  `https://collections.sentinel-hub.com/crop-biomass/open-collection.html`

[the remark this answers is not in the document]
Comment (Dorothyrono) [reply collections/crop-biomass.yaml:13]: We will add a tutorial to do a local build of the site,  to make it easier to check the pages before hand

[the remark this answers is not in the document]
Comment (mattballard-planet) [reply collections/crop-biomass.yaml:13]: ah, got it, I understand now. Let me go make these adjustments

[the remark this answers is not in the document]
Comment (mattballard-planet) [reply collections/crop-biomass.yaml]: okay, I think I've addressed each of these issues. Thanks for offering the tutorial, I wasn't sure how to do a local check. That might be helpful to double check some of the formatting before I push this

File: collections/crop-biomass.yaml (hunk 2 above)
[the remark this answers is not in the document]
Comment (mattballard-planet) [reply collections/crop-biomass.yaml]: fixed

File: collections/crop-biomass.yaml [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+  This enables better-informed decisions, input optimization, interventions, and harvest planning. 
+  This is a relative measure of biomass, so each pixel value has a value of 0 (low biomass) to 1 (high biomass). 
+  The product is provided daily and every day, the user gets a new raster with the data from the most updated direct observation from the day before.
+AdditionalInfoExternal:
+    Title: Open Data for Crop Biomass
+    Path: crop-biomass/open-data.md
+Documentation: "[here](https://docs.sentinel-hub.com/api/latest/data/planetary-variables/crop-biomass/)"
+Image: crop-biomass/crop-biomass.png
+Resolution: 10m
+GeographicalCoverage: Global; Open data in [limited areas](crop-biomass/open-data.html);
```
Comment (Dorothyrono) [inline collections/crop-biomass.yaml:13]: The root url is  `https://collections.sentinel-hub.com/crop-biomass/`. (should have mentioned this already)

Therefore, the link `(crop-biomass/open-data.html)`  is one level too high  

You just need  `(open-data.html)`.  That's it. 

You will see that if you do a local  build , see instructions on how to build the site on local [Link](https://github.com/sentinel-hub/public-collections/blob/main/CONTRIBUTING.md#how-to-build-sentinel-hub-public-collections-website-locally)

File: collections/crop-biomass.yaml [hunk 5]
```diff
@@ -0,0 +1,39 @@
...
+  - satellite imagery
+  - planetary variables
+  - agriculture
+  - biomass
+License: |
+  [License](https://www.planet.com/licensing-information/)
+Resources:
+  - EndPoint: services.sentinel-hub.com
+    Type: BYOC collection
+    Notes: Contains the data purchased by individual user. [Explore sample CC-BY-SA open data here](crop-biomass/open-data.html).
```
Comment (Dorothyrono) [inline collections/crop-biomass.yaml:32]: same comment as above and to the rest

**Review by Dorothyrono — CHANGES_REQUESTED** [review]

File: collections/crop-biomass.yaml [hunk 6]
```diff
@@ -0,0 +1,39 @@
+Name: Crop Biomass
+Description: |
+  [Crop Biomass](https://www.planet.com/products/planetary-variables/) provides the ability to track and respond to the highly dynamic nature of biomass in agricultural fields throughout the season. 
+  This enables better-informed decisions, input optimization, interventions, and harvest planning. 
+  This is a relative measure of biomass, so each pixel value has a value of 0 (low biomass) to 1 (high biomass). 
+  The product is provided daily and every day, the user gets a new raster with the data from the most updated direct observation from the day before.
+AdditionalInfoExternal:
+    Title: Open Data for Crop Biomass
+    Path: crop-biomass/open-data.md
```
[the remark this answers is not in the document]
Comment (mattballard-planet) [reply collections/crop-biomass.yaml:9]: yes, thanks, overlooked that, will do it now

**Review by Dorothyrono — APPROVED** [review]
All good

**Comment by Dorothyrono** [comment]
Should we merge it ?

or you can merge  it if you have the rights

**Comment by mattballard-planet** [comment]
@Dorothyrono I don't have the rights to merge it, but yes, I think this is good to go

**Pull request merged**